### PR TITLE
fix_502_and_505

### PR DESCRIPTION
**Pull request by Alexandru-Dumitru**
- fixed issue #502 in handler (variables had their definitions reversed)
- created integration test around `zowe jobs list jobs`
- generated new snapshot for `zowe jobs list -h` (#505)
- fixed command in shell script from `zowe jobs view` to `zowe jobs list`  (#505)

Signed-off-by: Alexandru-Dumitru <alexandru.dumitru@broadcom.com>

### CODE REVIEW

File: packages/zosjobs/src/cli/list/jobs/Jobs.handler.ts [hunk 1]
```diff
@@ -32,8 +32,8 @@ export default class JobsHandler extends ZosmfBaseHandler {
     public async processCmd(params: IHandlerParameters): Promise<void> {
 
         // Obtain the list of jobs - by default uses the session user and * for owner and prefix.
-        const prefix: string = (params.arguments.owner != null) ? params.arguments.owner : this.mSession.ISession.user;
-        const owner: string = (params.arguments.prefix != null) ? params.arguments.prefix : JobsConstants.DEFAULT_PREFIX;
+        const owner: string = (params.arguments.owner != null) ? params.arguments.owner : this.mSession.ISession.user;
```
Comment (tucker01) [inline packages/zosjobs/src/cli/list/jobs/Jobs.handler.ts]: Hey, can you take a look at `GetJobs.ts`... I think the call `GetJobs.getJobsByOwnerAndPrefix` is wrong here too? The method signature:
```
getJobsByOwnerAndPrefix(session: AbstractSession, owner: string, prefix: string)
```

Comment (Alexandru-Dumitru) [reply packages/zosjobs/src/cli/list/jobs/Jobs.handler.ts]: Hi @tucker01 . Thanks for spotting this. I've done the changes but pipeline is not running. Already raised the issue on slack.

**Comment by zFernand0** [comment]
@Alexandru-Dumitru @tucker01 
I thought issues marked as "good first issue" were intended for newcomers to look at them and try to make a PR with the fixes.
@MikeBauerCA or @JTonda, please correct me if I'm wrong.

**Comment by MikeBauerCA** [comment]
@zFernand0 that is the intent. However, it is great to see any issue fixed! Thank you @Alexandru-Dumitru!

**Comment by tucker01** [comment]
Hey, would you mind running `npm install` and then trying the integration tests (`npm run test:integration`)? I think they're all failing on the build because of an update to imperative. Might have to update the snapshots.

**Comment by Alexandru-Dumitru** [comment]
Hi all,
@tucker01 I updated the snapshots and now test passed :) 
@zFernand0 Sorry for this. I wanted to do some minor work on the CLI since it's been a while, and I noticed this issue so I thought I would also fix it. Also I was time constrained, so I picked an easy one :D

**Comment by tucker01** [comment]
It's great that you fixed this up... I'm sure we'll have more issues we can mark as "good first issue" as we go.

**Pull request merged**